src/app.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react"
+
+import {CoachRegistration} from "./components/CoachRegistration"
+import {Navbar} from "./components/Navbar"
+
+export interface AppState {loadedPage: string}
+
+export class App extends React.Component<{}, AppState> {
+    constructor(props: any) {
+        super(props)
+        this.state = {
+            loadedPage: "coachRegistration"
+        }
+        this.loadNewPage=this.loadNewPage.bind(this)
+    }
+
+    loadNewPage(pageToLoad: string): void {
+        this.setState({loadedPage: pageToLoad})
+    }
```

Comment:
If you use fat arrow functions you don't have to piss around binding this, it will automatically have the correct scope of this.